apps/prairielearn/src/ee/auth/azure/index.ts

```diff
@@ -13,7 +13,14 @@ export function getAzureStrategy() {
       cookieEncryptionKeys: config.azureCookieEncryptionKeys,
       loggingLevel: config.azureLoggingLevel,
       scope: ['openid', 'profile', 'email'],
-      responseType: 'code id_token',
+      // This should only be `id_token`, which will force the implicit flow to
+      // be used. When we specific `code id_token` here, it will use the hybrid
```

Comment:
Not sure if this is what you meant, but:
```suggestion
      // be used. When we specify `code id_token` here, it will use the hybrid
```